modules/seqexec/server/src/main/scala/seqexec/server/SeqexecEngine.scala

```diff
@@ -886,8 +889,10 @@ object SeqexecEngine extends SeqexecConfiguration {
         svs)
       case engine.BreakpointReached(id)                                    => SequencePaused(id,
         svs)
-      case engine.SingleRunCompleted(_, _)                                 => NullEvent
-      case engine.SingleRunFailed(_, _)                                    => NullEvent
+      case engine.SingleRunCompleted(ActionCoords(sid, _), res, _)         =>
+        SingleActionEvent(SingleActionOp.Completed(sid, res))
```

Comment:
But then this (which I just fixed and tested) still doesn't trigger the event sequence I need.